front/app/containers/Admin/projects/project/phase/phaseParticipationConfig/utils/participationMethodConfigs.ts

```diff
@@ -0,0 +1,108 @@
+import { Multiloc } from 'typings';
+
+import {
+  IdeaDefaultSortMethod,
+  InputTerm,
+  ParticipationMethod,
+  TSurveyService,
+  VotingMethod,
+} from 'api/phases/types';
+
+export interface IPhaseParticipationConfig {
```

Comment:
I've now introduced transparent participation method configuration, making it a lot easier to add new methods going forward.